packages/adminrouter/extra/dcos-adminrouter-agent-reload.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=Admin Router Reloader: Reload admin router to get new DNS
+Description=Admin Router Reloader: reload admin router to pick up domain resolution changes
```

Comment:
Instead of "admin router" let's use "Admin Router" (here, and in all other places in this diff).

That is the name we've tried to consistently use in the docs:
https://github.com/dcos/dcos-docs/search?utf8=%E2%9C%93&q=admin+router